front/src/pages/teachers/teachers-information/styles.js

```diff
@@ -0,0 +1,80 @@
+import style from 'styled-components';
+import {Col as col, Button as button, Form as form, Input as input} from 'antd';
```

Comment:
No need to convert uppercased components to lowercased

Reply:
Done